client/data/sites/site-excerpt-constants.ts

```diff
@@ -22,4 +22,5 @@ export const SITE_EXCERPT_REQUEST_OPTIONS = [
 	'updated_at',
 	'is_redirect',
 	'unmapped_url',
+	'admin_url',
```

Comment:
One thing we should definitely do before adding new data in the fields here is to make sure that the "full site" counterpart used in Calypso also includes it. The "excerpt" should always be a subset of the full site data, right?

Reply:
Yep that's right, otherwise we wouldn't be able to use the redux data as a quick way to get the initial data if it happens to exist.